content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,56 +2,26 @@
-that has a different resource requirement.
-
-A resize request is made through the pod `/resize` subresource, which takes the full updated pod for
-an update request, or a patch on the pod object for a patch request.
-
-For in-place resize of pod resources:
-- A container's resource `requests` and `limits` are _mutable_ for CPU
-  and memory resources. These fields represent the _desired_ resources for the container.
-- The `resources` field in `containerStatuses` of the Pod's status reflects the resources
-  _allocated_ to the pod's containers. For running containers, this reflects the actual resource
-  `requests` and `limits` that are configured as reported by the container runtime. For non-running
-  containers, these are the resources allocated for the container when it starts.
-- The `resize` field in the Pod's status shows the status of the last requested
-  pending resize. It can have the following values:
-  - `Proposed`: This value indicates that a pod was resized, but the Kubelet has not yet processed
-    the resize.
-  - `InProgress`: This value indicates that the node has accepted the resize
-    request and is in the process of applying it to the pod's containers.
-  - `Deferred`: This value means that the requested resize cannot be granted at
-    this time, and the node will keep retrying. The resize may be granted when
-    other pods are removed and free up node resources.
-  - `Infeasible`: is a signal that the node cannot accommodate the requested
-    resize. This can happen if the requested resize exceeds the maximum
-    resources the node can ever allocate for a pod.
-  - `""`: An empty or unset value indicates that the last resize completed. This should only be the
-    case if the resources in the container spec match the resources in the container status.
-
-If a node has pods with an incomplete resize, the scheduler will compute the pod requests from the
-maximum of a container's desired resource requests, and it's actual requests reported in the status.
+This page explains how to change the CPU and memory resource requests and limits assigned to a container *without recreating the Pod*.
 
+Traditionally, changing a Pod's resource requirements necessitated deleting the existing Pod and creating a replacement, often managed by a [workload controller](/docs/concepts/workloads/controllers/). In-place resizing allows containers within a running Pod to be allocated more or less CPU/memory, potentially avoiding application disruption.
+
+**Key Concepts:**
+
+* **Desired Resources:** A container's `spec.containers[*].resources` represent the *desired* resources for the container, and are mutable for CPU and memory.
+* **Actual Resources:** The `status.containerStatuses[*].resources` field reflects the resources *currently applied* for a running container. For containers that haven't started or were restarted, it reflects the resources allocated upon their next start.
```

Comment:
` currently applied for a running container` -> `currently configured on a running container`